### PR TITLE
Added Popup Alert to fill in survey in App.js

### PR DESCRIPTION
**Pull request by Bellex0**
I added a popup alert in App.js that reminds the user to fill in their health survey for today.

### CODE REVIEW

File: client/src/components/App.js [hunk 1]
```diff
@@ -21,11 +21,16 @@ const makeUserSession = () => {
 };
 
 class App extends Component {
+
   constructor(props) {
     super(props);
     this.userSession = new UserSession({ appConfig });
   }
 
+ popup = setTimeout(function() { alert("Fill in your health survey for today!") }, 5000)
```
Comment (SomeMoosery) [inline client/src/components/App.js]: Will this happy every time the app re-renders? I feel like a notification every 5 seconds or every time it re-renders would be too much? Or am I missing context?

Comment (Bellex0) [reply client/src/components/App.js]: Ok so I did git rebase master, hopefully it is good now. The notification is supposed to show up 5 seconds after you open the app. I was just thinking of this issue too. The function may need some specification stating that if the survey is filled out, then don't show this notification....

Comment (SomeMoosery) [reply client/src/components/App.js]: That makes sense. Looks like there's still merge conflicts. Did you push after rebasing?

Comment (Bellex0) [reply client/src/components/App.js]: I forgot to push 😣sorry about that! It should be fine now

File: client/yarn.lock [hunk 2]
```diff
@@ -1354,9 +1354,15 @@
...
 "@material-ui/lab@^4.0.0-alpha.46":
+<<<<<<< HEAD
+  version "4.0.0-alpha.47"
+  resolved "https://registry.yarnpkg.com/@material-ui/lab/-/lab-4.0.0-alpha.47.tgz#757a336e4c2496f700a392ff41e25a7d460a387b"
+  integrity sha512-+WC3O0M/769D3nO9Rqupusc+lob7tQMe5/DnOjAhZ0bpXlJbhZb7N84WkEk4JgQLj6ydP8e9Jhqd1lG+mGj+xw==
+=======
   version "4.0.0-alpha.46"
   resolved "https://registry.yarnpkg.com/@material-ui/lab/-/lab-4.0.0-alpha.46.tgz#453546d58d3a0064263c198dedb6a1c54d24d1a4"
   integrity sha512-JGgZmj1UNP8bbYNAGvndipjXRK3x2+9mFBzbX7MyCj+WpfnJbeqTmJK2No9MXvPj/EZJ1piaKif46FdDc4U93A==
+>>>>>>> upstream/master
```
Comment (SomeMoosery) [inline client/yarn.lock]: Looks like these merge conflicts wasn't fully resolved, the `<<<` `===` and `>>>` lines indicate a difference between incoming and current masters, I'd recommend reverting and trying the rebase / merge conflict resolution again

File: client/src/components/App.js [hunk 3]
```diff
@@ -21,9 +21,12 @@ const RADIKS_URL = process.env.REACT_APP_QA_URL || 'http://127.0.0.1:1260'; // T
 ReactBlockstack({ appConfig });
 
 function App(props) {
+
+  setTimeout(function() { alert("Fill in your health survey for today!") }, 5000)
+
   const { userSession } = useBlockstack();
   const finished = useCallback(({ userSession }) => {
-    if (RADIKS_URL) {
+    if (RADIKS_URL)
```
Comment (SomeMoosery) [inline client/src/components/App.js]: Does this work without the open bracket? Just curious, I know JavaScript is funky lol.

**Pull request merged**